sale_order_change_analytic_account/models/sale_order.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2021 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import _, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    def action_open_sale_order_change_analytic_wizard(self):
```

Comment:
@sbejaoui This function is not called anywhere.

Reply:
fixed